test/e2e/common/autoscaling_utils.go

```diff
@@ -58,9 +61,9 @@ const (
 )
 
 const (
-	KindRC         = "replicationController"
-	KindDeployment = "deployment"
-	KindReplicaSet = "replicaset"
+	KindRC         = "ReplicationController"
```

Comment:
This worked because during resource creation these constants were only used in a switch to go to the correct case, which then did not rely on them for choosing the proper API call. This is no longer the case, because now I use api.Kind for deletion.